### ISSUE TITLE
moble option kendogrid

### ISSUE THREAD

**Issue by johntom** [aurelia-ui-toolkits/aurelia-kendoui-bridge#778]
Is the mobile option supported for resoonsive display and if so what version. While it's been a while since I asked a question, I still use this library.
Hope all is well and thanks,
John

**Comment by johntom** [comment]
Yea, I tried that with the following   k-mobile="phone",  k-mobile="true", k-mobile.bind etc but grid still not responsive. On another note I've neen doing all recent app developement using front-end-creator and extremely happy with results.

**Comment by JeroenVinke** [comment]
Hey John, k-mobile.bind="true" should be the one. What I'd recommend is setting that attribute on the ak-grid, and then open the chrome console. After the grid loads you will see a log message in there containing the actual configuration object that was used to initialize the grid. You can check if that property is there.

If not, a bindable is probably missing, that could happen if this mobile property is new. You can work around this with `<ak-grid k-options.bind="{ mobile: true }"`

If the property is on there then the bridge did its job properly. You might have to contact kendo support to see what's up. Only thing I could think of is the grid being initialized on a fixed size div. If you do 

```
<ak-grid>
<ak-col></ak-col>
<div style="some css"></div>
<ak-grid/>
```

In other words, if you have a div tag in the ak grid then the bridge will initialize the grid on that div, if you don't the grid adds a div in the ak grid itself and initializes on it. But if you add it yourself you can control the css of the div
